selfdrive/car/hyundai/carcontroller.py

```diff
@@ -62,24 +65,48 @@ def update(self, enabled, CS, frame, actuators, pcm_cancel_cmd, visual_alert,
                         left_lane, right_lane, left_lane_depart, right_lane_depart)
 
     can_sends = []
+
+    # tester present - w/ no response (keeps radar disabled)
+    if CS.CP.openpilotLongitudinalControl:
+      if (frame % 10) == 0:
+        can_sends.append([0x7D0, 0, b"\x02\x3E\x80\x00\x00\x00\x00\x00", 0])
```

Comment:
slow this down to 2 Hz or even 1 Hz? only needs to be once every 5 seconds, and diagnostic tools generally send it at 1 Hz